entity/src/main/java/com/wilma/entity/positions/PositionCategory.java

```diff
@@ -0,0 +1,30 @@
+package com.wilma.entity.positions;
+
+import com.wilma.entity.users.Partner;
+
+import javax.persistence.Entity;
+import javax.persistence.Column;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.Period;
+import java.util.Date;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@Table(name = "position_category")
+public class PositionCategory extends Position {
```

Comment:
There's 2 things I'm not following, but I'm sure that's due to me not viewing the star UML file, so I'm not making a thing of it... Simply stating my thoughts
1. I don't understand how a category can be a position, eg **applying for a category** doesn't make sense to me.
2. I don't follow why we would need a category for positions separately to forum categories, I thought we just needed a category

Reply:
Job Categories should be the basis - forum categories should reflect that. We are offering jobs, forums are catering to questions regarding those jobs.

Reply:
So I need to change [Category.java](https://github.com/nick-mcguffin/Application-Development-Project/blob/ADP-78-forum-page/entity/src/main/java/com/wilma/entity/Category.java) to ForumCategory.java?